apache/uima-uimafit
# [UIMA-6448] Remove binary release ZIP build from uimaFIT

**JIRA Ticket:** https://issues.apache.org/jira/browse/UIMA-6448

**What's in the PR**
- Removed binary ZIP/tar assembly and associated LICENSE/NOTICE files

**How to test manually**
* Check if build runs with `-Papache-release` and that no binary ZIP/tar is created anymore

**Automatic testing**

**Documentation**

**Organizational**
      <sub><sup>Only dependencies under [approved licenses](http://www.apache.org/legal/resolved.html#category-a) are allowed. LICENSE and NOTICE files in the respective modules where dependencies have been added as well as in the project root have been updated.</sup></sub>
